anomalib/deploy/inference.py

```diff
@@ -80,19 +80,22 @@ def predict(
             np.ndarray: Output predictions to be visualized.
         """
         if meta_data is None:
-            meta_data = {}
+            if hasattr(self, "meta_data"):
+                meta_data = getattr(self, "meta_data")
+            else:
+                meta_data = {}
         if isinstance(image, str):
             image = read_image(image)
         meta_data["image_shape"] = image.shape[:2]
 
         processed_image = self.pre_process(image)
         predictions = self.forward(processed_image)
-        anomaly_map, pred_score = self.post_process(predictions, meta_data=meta_data)
+        anomaly_map, _ = self.post_process(predictions, meta_data=meta_data)
 
         if superimpose is True:
             anomaly_map = superimpose_anomaly_map(anomaly_map, image)
 
-        return anomaly_map, pred_score
+        return anomaly_map
```

Comment:
In OTE, the OpenVINO task expects both the anomaly map and the predicted score to be returned. We'll need to make some changes there to make it compatible with this design.